api/apps/search/queries.coffee

```diff
@@ -0,0 +1,19 @@
+module.exports =
+  matchAll: (term) ->
+    {
+      bool: {
+        must: {
+          multi_match: {
+            query: term,
+            type: 'best_fields',
+            fields: ['name.*', 'alternate_names.*'],
+            fuzziness: 2
```

Comment:
As I mentioned, `AUTO` will default to 2 for strings of more than five characters, as per [the doc](https://www.elastic.co/guide/en/elasticsearch/guide/current/fuzziness.html). But forcing it to two is good. You can't go higher on ES 2.x afaik.

Reply:
👍